app/controllers/concerns/paginable.rb

```diff
@@ -152,67 +130,62 @@ def refine_query(scope)
     scope
   end
 
+  def sort_direction
+    @sort_direction ||= SortDirection.new(@paginable_params[:sort_direction])
+  end
+
   # Returns the sort link name for a given sort_field. The link name includes
   # html prevented of being escaped
   def sort_link_name(sort_field)
     class_name = "fa-sort"
     if @paginable_params[:sort_field] == sort_field
-      class_name = upcasing_sort_direction == "ASC" ? "fa-sort-asc" : "fa-sort-desc"
+      class_name = "fa-sort-#{sort_direction.downcase}"
     end
     <<~HTML.html_safe
       <i class="fa #{class_name}"
          aria-hidden="true"
          style="float: right; font-size: 1.2em;">
 
         <span class="screen-reader-text">
-
-        Sort by #{sort_field.split(".").first}
+          Sort by #{sort_field.split(".").first}
```

Comment:
We should probably wrap this for FastGetText `_("Sort by #{sort_field...}")`.

Good catch on the missing `</span>`